processor/src/decoder/trace.rs

```diff
@@ -513,6 +517,18 @@ impl DecoderTrace {
         self.group_count_trace.push(ZERO);
         self.op_idx_trace.push(ZERO);
     }
+
+    /// Fetches all the helper registers from the trace.
+    #[cfg(test)]
+    pub fn user_helpers(&self) -> [Felt; TOTAL_USER_OP_HELPERS] {
```

Comment:
Is there a reason this is named `user_helpers()` but the on in the decoder is `get_user_helpers()`?

In general, to keep naming consistent, I'd prefer both to be named `get_user_op_helpers()`.

Reply:
done